### PR TITLE
LAST CALL - READY to add eip155

### PR DESCRIPTION
**Pull request by bumblefudge**
Open question:
- should the JSON-RPC specs be incorporated as an ## CAIP25 and/or ## CAIP27 section?
   + on 2/10 call, sections purely for examples might be useful but overly verbose- might make sense to generalize CAIP25 and/or 27 to define better how those sections get added to namespaces
- does the intro need some work? I feel like I'm probably glossing over many subtleties of the difference between `evm` and `eip155` 😅

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Initial thoughts as comments
Also guess when adding more namespaces in the PR we might see more clearly where duplications for for certain sections would happen.

File: eip155.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+*For context, see the [CAIP-2][] specification.*
+
+### Rationale
+
+The chain ID defined in EIP155 is the most widely used chain identifier in the
+Ethereum ecosystem known to the authors. It strives for uniqueness and the fact
+that the standard is used for replay protection ensure that creators of a new
+Ethereum network have an incentive to use an ID that is not used elsewhere.
+These new Ethereum networks can be self-registered in the [chainid.network][]
```
[the remark this answers is not in the document]
Comment (bumblefudge) [reply eip155.md:37]: Makes sense!

File: eip155.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+## CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
+
+### Rationale
+
+While Ethereum "accounts" were the unstated norm in the definition of
+[CAIP-10][], there are still some particularities of the syntax that should be
+specified for the unfamiliar.  
+
```
[the remark this answers is not in the document]
Comment (bumblefudge) [reply eip155.md:92]: I reworked it to be a ### Context instead, maybe that's clearer?

File: eip155.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+namespace-identifier: eip155
+title: EIP155 Namespace, aka EVM Chains
+author: Simon Warta (@webmaster128), ligi <ligi@ligi.de>, Pedro Gomes (@pedrouid), Antoine Herzog (@antoineherzog), Pedro Gomes (@pedrouid), Joel Thorstensson (@oed)
+discussions-to: https://github.com/ChainAgnostic/namespaces/pulls/2
+status: Draft
+type: Standard
+created: 2022-01-19
+updated: 2022-01-19
+requires: CAIP-2, CAIP-10, CAIP-19
+supercedes: CAIP-3, CAIP-21, CAIP-22
```
Comment (ligi) [inline eip155.md:11]:
```suggestion
supersedes: CAIP-3, CAIP-21, CAIP-22
```

File: eip155.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+This document describes the syntax and structure of the [EIP155][] namespace,
+commonly referred to as the "EVM" namespace for chains using the Ethereum
+Virtual Machine. 
+
+## Introduction
+
+[EIP155][] is an Ethereum Improvement Process specifying addressing across
+Ethereum-based chains, generalized by [CAIP-2][]. Definition of valid
+`references` within that namespace and constraints are deferred to the
+[EIP155][] specification, which may be superceded by future EIPs.
```
Comment (ligi) [inline eip155.md:25]:
```suggestion
[EIP155][] specification, which may be superseded by future EIPs.
```

**Review by ligi — CHANGES_REQUESTED** [review]
Just a typo - but think it is ready to merge soon - thanks for your work!

image

**Review by ligi — APPROVED** [review]

**Review by oed — COMMENTED** [review]
Looks good. The only thing I would change is the name of the files, e.g. `eip155/caip10.md` instead of `eip155/addresses.md`. Would also be nice to have links to these documents from the `eip155/README.md`.

**Comment by oed** [comment]
Oh, I see your comment now. I still think `caip10.md` would be better than `addresses.md` because it's clear what it refers to. And it will be easier to see which CAIPs a namespace supports.

**Comment by bumblefudge** [comment]
That last point won me over, caip-2.md it is! 😄 

As for readmes:
- I think when viewed in Github or, say, VSC, the file explorer will function as a TOC, non?
- Once I'm done building out v1 of /namespaces/ i'll start looking at a way to automate github pages using the jekyll-style YAML frontmatter... surely there's a way to autogenerate TOCs at the root and per-namespace level?

File: eip155/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## References
+
+- [EIP155][]: Ethereum Improvement Proposal specifying generation and validation of ChainIDs
+- [ethereum-lists/chains][]: An open registry for eip155 network operators to claim a
+      unique chainID and self-publish RPC/node information for them.
+- [ERC20][]: Basic [aka Fungible] Token Standard
+- [ERC721][]: Non-Fungible Token Standard
+
+[Chainid.network]: https://github.com/ethereum-lists/chains
```
Comment (oed) [inline eip155/README.md:27]:
```suggestion
[Chainid.network](https://github.com/ethereum-lists/chains)
```

File: eip155/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+Virtual Machine. 
+
+## Introduction
+
+[EIP155][] is an Ethereum Improvement Process specifying addressing across
+Ethereum-based chains, generalized by [CAIP-2][]. Definition of valid
+`references` within that namespace and constraints are deferred to the
+[EIP155][] specification, which may be superseded by future EIPs.
+
+## References
```
Comment (oed) [inline eip155/README.md:19]: I think the markdown of this section is not correctly formatted.

File: eip155/caip10.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+type: Standard
+created: 2022-01-19
+updated: 2022-01-19
+requires: CAIP-2, CAIP-10
+supersedes: CAIP-3
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
```
Comment (oed) [inline eip155/caip10.md:16]:
```suggestion
*For context, see the [CAIP-10]() specification.*
```
And similar across the spec.

**Review by oed — CHANGES_REQUESTED** [review]
Structure lgtm, some fixes needed to markdown formatting.

File: eip155/README.md (hunk 5 above)
Comment (bumblefudge) [reply eip155/README.md:27]: check it in github, this is a cool GH-flavored markdown trick that cel taught me!

File: eip155/README.md (hunk 6 above)
Comment (bumblefudge) [reply eip155/README.md:19]: see below! the first part is a bulleted list, the second part is invisible in the rendered markdown

File: eip155/README.md (hunk 5 above)
Comment (oed) [reply eip155/README.md:33]: Oh, I see. That's confusing :)

**Review by oed — APPROVED** [review]

**Comment by bumblefudge** [comment]
2 approvals, should we 🚢 it, @ligi ?

**Pull request merged**